test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 2950
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true", ]
+stress_cmd: [
+  "bin/ycsb run  dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p operationcount=535000000 -p dataintegrity=true", ]
+
+dynamodb_primarykey_type: HASH_AND_RANGE
+
+n_loaders: 3
+instance_type_db: 'i3.4xlarge'
+n_monitor_nodes: 1
+n_db_nodes: 4
+
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+nemesis_during_prepare: true
+
+user_prefix: 'alternator-48h'
+
+alternator_port: 8080
```

Comment:
perhaps we can have a generic yaml file with alternator general defaults like alternator_port, alternator_access_key_id, etc..

Reply:
The only way to enable Alternator is to set the alternator port, so we couldn't use it in a default file, otherwise Alternator will be always enabled.

Reply:
@yarongilor we are following all other configuration, like cql user/password.
we want it visible in the yaml, what this one is doing.